tests/build_disk_index.cpp

```diff
@@ -41,6 +42,9 @@ int main(int argc, char **argv)
         desc.add_options()("num_threads,T", po::value<uint32_t>(&num_threads)->default_value(omp_get_num_procs()),
                            "Number of threads used for building index (defaults to "
                            "omp_get_num_procs())");
+        desc.add_options()("QD", po::value<uint32_t>(&QD)->default_value(0), " Quantized Dimension for compression");
```

Comment:
let's allow only one of QD or B parameters?

Reply:
> let's allow only one of QD or B parameters?

same reason above.